tractor/_actor.py

```diff
@@ -404,17 +444,23 @@ async def _push_result(
         send_chan, recv_chan = self._cids2qs[(actorid, cid)]
         assert send_chan.cid == cid  # type: ignore
 
-        if 'stop' in msg:
-            log.debug(f"{send_chan} was terminated at remote end")
-            # indicate to consumer that far end has stopped
-            return await send_chan.aclose()
+        # if 'stop' in msg:
```

Comment:
yeah i think we can drop this now.
processing is now inside the `ReceiveMsgStream.receive()` body.